game/script-topics.rpy

```diff
@@ -599,6 +599,573 @@ label talk_eating_well:
+    n 1nlrsm "..."
+    n 1nnmss "Did you get there okay, [player]?"
+
+    menu:
+        "Yes, I have the website open.":
+            n 1nchbg "Awesome!{w=0.2} Step one complete!"
+
+        "No, I couldn't get to the website.":
+            n 1tnmpu "Huh?{w=0.2} Why not?{w=0.2} Is it down or something?"
+            n 1nnmss "Well, anyway...{w=0.3} maybe we can try this again later?"
+            n 1nnmbg "Just let me know when you're ready,{w=0.1} 'kay?"
+            return {"lock" : None}
+
+    # Prompt the player to create an account
+
+    n 1nnmss "'Kay!{w=0.2} Now for step two!"
+    n 1nnmaj "Basically I need something called an API key,{w=0.1} which will let me use that website to find out what the weather is like over there."
+    n 1nlraj "But I can't do that myself...{w=0.3} which is where you come in,{w=0.1} [player]!"
+    n 1nnmss "You'll need to make an account before you can get an API key though."
+    n 1nwrbg "Don't worry{w=0.1} -{w=0.1} it's totally free,{w=0.1} I promise!"
+    n 1nnmss "You can create an account {a=[store.jn_globals.LINK_OPEN_WEATHER_MAP_SIGN_UP]}here{/a},{w=0.1} or you can sign in using the menu at the top."
+    n 1fwdaw "Just make sure to go through all the options carefully{w=0.1} -{w=0.1} don't just dash through it!"
+    n 1nllaj "Oh{w=0.1} -{w=0.1} and make sure you confirm your email address once you've created it,{w=0.1} 'kay?"
+    n 1nnmss "{a=[store.jn_globals.LINK_OPEN_WEATHER_MAP_SIGN_UP]}Here's{/a} that link once more,{w=0.1} just in case!{w=0.2} Just talk to me again when you're done."
+    n 1nwmsm "..."
+    n 1nnmaj "Well,{w=0.1} [player]?{w=0.2} All good?"
+
+    menu:
+        "Yes, I have an account set up.":
+            n 1nchbg "Nice work,{w=0.1} [player]!"
+            n 1nnmss "You'll probably want to make sure you save your login details somewhere secure,{w=0.1} just in case."
+            n 1nnmaj "I hope you didn't forget to confirm your email address too!"
+            n 1nnmss "Alright!{w=0.2} Now here's the challenging part."
+
+        "I already had an account set up.":
+            n 1uwdaj "Oh?{w=0.2} I didn't realise you were such a pro at this,{w=0.1} [player]!{w=0.2} Ehehe."
+
+            if already_discussed_setup:
+                n 1nlrdv "Or maybe we went through this before,{w=0.1} hmm?" #Not quite sure with this expression
+
+            n 1nnmbg "Well, anyway.{w=0.2} The rest of this should be a piece of cake then!"
+
+        "Never mind.":
+            n 1tnmpu "Huh?{w=0.2} You don't wanna continue?"
+            n 1nlraj "That's fine,{w=0.1} I guess."
+            n 1nnmss "Just let me know when you're ready,{w=0.1} 'kay?"
+            return {"lock" : None}
+
+    # Prompt the player for an API key
+
+    n 1nnmbg "Are you ready,{w=0.1} [player]?"
+    n 1nllss "You need to get your API key and send it to me!"
```

Comment:
Natsu looks a little awkward here - I'd suggest something like `1uchbg` to match her intonation? (As the line comes off as triumphant/confident).